### PR TITLE
Bump ksonnet reference to 0.11.0

### PR DESCRIPTION
**Pull request by pdmack**
Should be OK to point users at the latest.
<fingers-crossed>

/assign @abhi-g 
/assign @ewilderj

### CODE REVIEW

File: content/docs/user_guide.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ For an end to end example illustrating in details how to deploy kubeflow and run
 
 ## Requirements
  * Kubernetes >= 1.8 [see here](https://github.com/kubeflow/tf-operator#requirements)
- * ksonnet version [0.9.2](https://ksonnet.io/#get-started). (See [below](#why-kubeflow-uses-ksonnet) for an explanation of why we use ksonnet)
+ * ksonnet version [0.11.0](https://ksonnet.io/#get-started). (See [below](#why-kubeflow-uses-ksonnet) for an explanation of why we use ksonnet)
```
Comment (abhi-g) [inline content/docs/user_guide.md]: @kunmingg Can you please take a look. 
We could also say ksonnet >= 0.x.x depending on what's the bare minimum required. If it's not critical, I am okay with switching to 0.11.0

Comment (kunmingg) [reply content/docs/user_guide.md]: 0.11.0 should be good, thanks!

**Pull request merged**